rules/rules-reviewed/eap8/eap6/eap8-xml-binding.windup.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="eap8-xml-binding"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis of XML Binding for migration to JBoss EAP 8.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="eap" versionRange="[6,7]" />
+        <targetTechnology id="eap" versionRange="[8,)" />
+        <tag>eap8</tag>
+    </metadata>
+    <rules>
+        <rule id="eap8-xml-binding-00001">
+            <when>
+                <or>
+                    <javaclass references="com.sun.xml.bind.api.{*}">
+                        <location>IMPORT</location>
```

Comment:
What if the class is used with the full qualified name instead of importing? Maybe it's better to remove the `IMPORT` location in this one and similar ones?